build/common.props

```diff
@@ -18,8 +17,8 @@
     <RepositoryType>git</RepositoryType>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=2028464</PackageLicenseUrl>
-    <PackageIconUrl>https://raw.githubusercontent.com/Azure/azure-webjobs-sdk/dev/webjobs.png</PackageIconUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
These changes won't be checked in - had to make them temporarily to get local package build working.

Reply:
You might want to make this change at some point. `<PackageLicenseUrl>` and `<PackageIconUrl>` have been long deprecated.

Reply:
Agreed